urld5c972ef81_noxx90_jflam_tgz-pJ8-fr_noxx90_jflam_BenchmarkJ8/src/fr/noxx90/jflam/FlameLogger.java

```diff
@@ -0,0 +1,63 @@
+package fr.noxx90.jflam;
+
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.esotericsoftware.minlog.Log.Logger;
+
+public class FlameLogger extends Logger {
+	private SimpleDateFormat format;
+	private PrintWriter writer;
+	
+	public FlameLogger() {
+		format = new SimpleDateFormat("HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.